content/en/service_management/app_builder/auth.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Access and Authentication
+kind: documentation
+description: Access and authentication for App builder
+---
+
+{{< callout url="https://www.datadoghq.com/dg/datadog-app-builder/" btn_hidden="false" header="Join the Beta!">}}
+Datadog App Builder is in private beta. Complete the form to request access.
+{{< /callout >}}
+
+A few tools control access and authentication for apps and their components.
+
+## App identity
```

Comment:
we should remove this `App identity` section. it doesn't apply to apps the same way it applies to workflows. I will add clarity below but I think it's covered elsewhere, so we can just rid of this section. 

--- 

more detail: apps aren't run as the user that created them. the queries in apps are run when the app is loaded, and those queries leverage connections. users must have resolver permissions on the appropriate connections. if they don't, the queries won't run.

Reply:
Happy to remove that section

Reply:
Thank you, I'll remove it.